src/Symfony/Component/AssetMapper/Command/AssetMapperCompileCommand.php

```diff
@@ -74,6 +73,26 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $entrypointFiles[$entrypointName] = $path;
         }
 
+        if ($this->isDebug && !$input->getOption('force')) {
+            $didUnlink = false;
+            $publicDir = $this->assetsFilesystem->getDestinationPath();
+
+            foreach ($this->assetMapper->allAssets() as $asset) {
+                if (is_file($publicDir.'/'.$asset->publicPath)) {
+                    unlink($publicDir.'/'.$asset->publicPath);
+                    $didUnlink = true;
+                }
+            }
+
+            if ($didUnlink) {
+                $io->warning('Running in debug mode: removed compiled assets to let them be served dynamically. Use --force to generate them instead.');
+            }
+
+            return 0;
```

Comment:
Does the private class method createManifestAndWriteFiles() will be call somewhere ? It seems files will not be compiled again :thinking:

Reply:
This is inside the condition `if ($debug && !$force)`. We won't indeed compile assets in that case, as the debug mode is meant to compile files on-demand when they don't exist on disk (if you actually want to dump generated files while in debug mode, that's what `--force` is about)

Reply:
You are right I miss the "NOT" force :pray: